docs/accelerometer-LIS3DH.md

```diff
@@ -77,7 +77,7 @@ board.on("ready", function() {
 ## License
 Copyright (c) 2012, 2013, 2014 Rick Waldron <waldron.rick@gmail.com>
 Licensed under the MIT license.
-Copyright (c) 2017 The Johnny-Five Contributors
+Copyright (c) 2018 The Johnny-Five Contributors
```

Comment:
I didn't see any changes in this commit besides the copyright year change. Did I miss something?

Reply:
No: The CI setup for this repo regenerates these docs, and complains if it finds differences between what it generated and the committed copies.  Due to the year change, it's requiring me to commit updates for all the documentation files to pass CI.